2.11.0/nfs-operator.yaml

```diff
@@ -0,0 +1,186 @@
+# This manifest deploys the OpenEBS control plane components, with associated CRs & RBAC rules
```

Comment:
Q: Since dynamic-nfs-provisioner is not yet declared as beta, do we need to add nfs-operator.yaml under 2.11.0

Reply:
It is ok to add the alpha yamls as well. Just an easy way to get access to them.